lib/src/exceptions.dart

```diff
@@ -6,7 +6,7 @@ class KeyNotFoundException implements Exception {
   final String message;
 
   /// Takes the [map] and [key] and creates the [message]
-  KeyNotFoundException(Map<dynamic, dynamic> map, dynamic key)
```

Comment:
Should the dynamic change here be `Object?`. A key could be null, the value in a map could be null